packages/exchange/src/exchange/langfuse_wrapper.py

```diff
@@ -14,29 +14,26 @@
 
 import os
 from typing import Callable
-from dotenv import load_dotenv
 from langfuse.decorators import langfuse_context
 import sys
 from io import StringIO
-from pathlib import Path
-from functools import wraps  # Add this import
+from functools import cache, wraps  # Add this import
 
-
-def find_package_root(start_path: Path, marker_file: str = "pyproject.toml") -> Path:
-    while start_path != start_path.parent:
-        if (start_path / marker_file).exists():
-            return start_path
-        start_path = start_path.parent
-    return None
+DEFAULT_LOCAL_LANGFUSE_HOST = "http://localhost:3000"
+DEFAULT_LOCAL_LANGFUSE_PUBLIC_KEY = "publickey-local"
```

Comment:
nit: maybe add a comment here that the public/private key defaults come from the .env.langfuse.local file. As these are what langfuse is locally initialized with as defined there. These values are not actually default from Langfuse itself.